src/main/java/org/jabref/gui/linkedfile/DeleteFileAction.java

```diff
@@ -116,7 +116,32 @@ private void deleteFiles(List<LinkedFileViewModel> toBeDeleted, boolean deleteFr
      */
     public void deleteFileFromDisk(LinkedFileViewModel fileViewModel) {
         LinkedFile linkedFile = fileViewModel.getFile();
+        deleteFileHelper(dialogService, preferences, databaseContext, linkedFile);
+    }
+
+    /**
+     * Deletes files from disk without asking the user for confirmation, for a list of files
+     *
+     * @param dialogService DialogService
+     * @param preferences PreferencesService
+     * @param databaseContext BibDatabaseContext
```

Comment:
You can delete this - no new information is given. - Similar for other "empty" JavaDocs